test/test_evaluation/evaluation_util.py

```diff
@@ -107,10 +109,26 @@ def get_multiclass_classification_datamanager():
 
 
 def get_abalone_datamanager():
-    dataset = 'abalone'
-    dataset_path = os.path.join(os.path.dirname(__file__), '.datasets',
-                                dataset)
-    D = CompetitionDataManager(dataset_path)
+    # https://www.openml.org/d/183
+    dataset_name = 'abalone'
+    task = 'multiclass.classification'
```

Comment:
Shouldn' this be a constant from [here](https://github.com/automl/auto-sklearn/blob/master/autosklearn/constants.py)?

Reply:
Did you have a look at this?